Sources/Passes/Passes.docc/GettingStarted.md

```diff
@@ -0,0 +1,365 @@
+# Getting Started with Passes
+
+Create the pass data model, build a pass and distribute it.
```

Comment:
We should probably tweak this, if not for SEO than anything else. It would be good to mention that this is to provide a PassKit server implementation